JENKINS-HOME-TEMPLATE/init.groovy.d/02_plugins.groovy

```diff
@@ -4,7 +4,7 @@ import hudson.model.*
 import java.util.logging.Logger
 Logger log = Logger.getLogger('init.groovy.d')
 
-def plugins = ["workflow-aggregator", "filesystem_scm", "git", 'pipeline-utility-steps', 'http_request']
+def plugins = ["workflow-aggregator", "filesystem_scm", "git", 'pipeline-utility-steps', 'http_request', 'docker-plugin', 'docker-workflow']
```

Comment:
why is the init.groovy.d still used? Is the jenkins not hosted in a docker container? Is it on bare metal?

Reply:
Hi Dane,
our lab jenkins does not run inside docker, see 

I still use init.groovy.d for my local dev environment see 
https://github.com/baloise/CICD-pipeline/blob/feat/Maven/docs/decisions/006-homogeneous-build-across-environments.md
Just discovered that docker shared volumes now work on our windows boxes, so we can replace the maven - based dev environment with docker.
On the other hand we use maven anyhow for classpath management in the IDE.